src/test/java/org/isf/medicalstock/Tests.java

```diff
@@ -229,7 +229,7 @@ void testIoGetLotsByMedical(boolean in, boolean out, boolean toward) throws Exce
 		int code = setupTestMovement(false);
 		Movement foundMovement = movementIoOperationRepository.findById(code).orElse(null);
 		assertThat(foundMovement).isNotNull();
-		List<Lot> lots = medicalStockIoOperation.getLotsByMedical(foundMovement.getMedical());
+		List<Lot> lots = medicalStockIoOperation.getLotsByMedical(foundMovement.getMedical(), true);
```

Comment:
All the tests appear to use the `true` parameter; are there any that use `false`?

Reply:
yes I added